backend/local_llamastack_server/README.md

```diff
@@ -0,0 +1,27 @@
+# Local Llama Stack Server
```

Comment:
I'd suggest we have support for locall testing in a directory called `local_testing` at the top level. As mentioned before a config file which has the environment variables that are set in the helm deployment and a script to set them in the enviroment would be good.